wallet/udb/txdb.go

```diff
@@ -547,6 +547,22 @@ func putRawTxRecord(ns walletdb.ReadWriteBucket, k, v []byte) error {
 	return nil
 }
 
+func readRawTxRecordMsgTx(txHash *chainhash.Hash, v []byte, msgTx *wire.MsgTx) error {
+	if len(v) < 8 {
+		str := fmt.Sprintf("%s: short read for raw tx record msg tx(expected %d "+
+			"bytes, read %d, txHash %v)", bucketTxRecords, 8, len(v), txHash)
+		return storeError(apperrors.ErrData, str, nil)
```

Comment:
I'd like to use apperrors.New/Wrap for new instances.  Eventually we can replace all instances of storeError with that.